src/common/services/ApiService.ts

```diff
@@ -195,13 +195,13 @@ export default class ApiService {
   }
 
   static registerTx({
-    txHash, type, value, wallet, fee,
+    sessionId, txHash, type, value, wallet, fee,
   }: {
-    txHash: string, type: string, value: number, wallet: string, fee: number,
+    sessionId: string, txHash: string, type: string, value: number, wallet: string, fee: number,
   }): Promise<void> {
     return new Promise<void>((resolve, reject) => {
       axios.post(`${ApiService.baseURL}/register`, {
```

Comment:
avoid to call /register endpoint it some of the parameters are null for example:

if (sessionId === null || txHash === null || type === null )
 return;